plugins/systemd/systemd.plugin.zsh

```diff
@@ -20,3 +20,14 @@ alias sc-mask-now="sc-mask --now"
 alias scu-enable-now="scu-enable --now"
 alias scu-disable-now="scu-disable --now"
 alias scu-mask-now="scu-mask --now"
+
+function sc-prompt-status {
+  for UNIT in $@
+  do
+    [ -n "$ZSH_THEME_SYSTEMD_PROMPT_CAPS" ] && UNITDISPLAY=$(tr '[:lower:]' '[:upper:]' <<< $UNIT) || UNITDISPLAY=$UNIT
```

Comment:
You can use `${(U)var}` to uppercase a variable without forking to `tr`.
Also, use `[[` instead of `[`.